qmetaobject/main.qml

```diff
@@ -0,0 +1,3 @@
+import QtQuick 2.0
+Item {}
+// This file exists only for doctests
```

Comment:
I too, don't like it. But from documentation perspective, it produces cleaner minimal doctest code, i.e. does not require extra subdirectories in path. Unfortunately, there's no way of changing cwd for `cargo test` command. So, a trade-off has to be made.

Please, confirm if you insist on moving files out of create's root directory.

Reply:
I don"r insist. But i still would like this to be clean.

Can you make it like this:

```rust
qrc!(my_resource,
     "foo" {
         "qrc_example/main.qml",
         "qrc_example/Bar.qml" as "baz/Foo.qml",
￼      }
￼);
```

otherwise, just leave the comment as ```ignore,  it is should still be tested by other tests.

We could also imagine to pass a path in the qrc! macro,  and it would be hidden with a # for the doc.
```rust
qrc!(my_resource,
# #[path("../tests/qrc_test")]
     "foo" {
```

But i don't want to insist.

Reply:
I like those fresh ideas out of nowhere.

Talking about macro syntax expansion, how about being consistent with file definitions, but at the cost of not being consistent with Qt's *.qrc format?

```rust
qrc!(my_resource,
     "./qrc_example" as "foo" {
         "main.qml",
         "Bar.qml" as "baz/Foo.qml",
      }
);
```

so here,  "./qrc_example" as "foo"  meas directory "./qrc_example" is used as a base path for stuff under prefix "foo".

Reply:
> so here, "./qrc_example" as "foo" meas directory "./qrc_example" is used as a base path for stuff under prefix "foo".

looks good

Reply:
Implemented in 26a0572.